### PR TITLE
Add historical balance metric

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: test/integration/sanHistoricalBalance.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    expect(headers).to.deep.equal(expectedHeaders)
+  })
+
+  it('returns a record per every day', () => {
+    const results = san.SAN_HISTORICAL_BALANCE(slug, from, to, address)
+
+    assertNumberOfRecords(results, numberOfDays)
+
+    for (let [index, day] of days.entries()) {
+      expect(results[index + 1][0]).to.equal(formatDate(day))
```
Comment (IvanIvanoff) [inline test/integration/sanHistoricalBalance.js]: can we test that we have non negative number values also?

Comment (spiderjako) [reply test/integration/sanHistoricalBalance.js]: Okay, I will add that

File: test/integration/sanHistoricalBalance.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  address
+} = require('../setup.js')
+
+describe('SAN_HISTORICAL_BALANCE', () => {
+  const expected = { date: 'string', balance: 'number' }
+
+  const response = san.SAN_HISTORICAL_BALANCE(slug, from, to, address)
+  const headers = response[0]
+  const results = response[1]
+  console.log(response)
```
Comment (kalkov) [inline test/integration/sanHistoricalBalance.js]: leftover

File: test/setup.js [hunk 3]
```diff
@@ -20,6 +20,8 @@ const days = eachDay(from, subDays(to, 1)) // last day should not be included (h
 const historicDataTo = subDays(endOfYesterday(), 200)
 const historicDataFrom = startOfDay(subDays(historicDataTo, 205))
 
+const address = '0x1f3df0b8390bb8e9e322972c5e75583e87608ec2'
```
Comment (kalkov) [inline test/setup.js]: can you rename to something more descriptive like `eth_address`?

File: src/api/queries/blockchainQueries.js [hunk 4]
```diff
@@ -158,3 +158,20 @@ ApiClient_.prototype.fetchRealizedValue = function (slug, from, to) {
 
   return this.conn.graphQLQuery(query, 'realizedValue')
 }
+
+ApiClient_.prototype.fetchHistoricalBalance = function (slug, from, to, address) {
+  var query = {
+    'query': '{\
+       historicalBalance(slug: "' + slug + '",\
+                     from: "' + toUTC_(from) + '",\
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: indent at the same level as `slug`

File: test/setup.js (hunk 3 above)
Comment (spiderjako) [reply test/setup.js]: Alright, I'll do it

File: src/api/queries/blockchainQueries.js (hunk 4 above)
Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: I will change it, thanks

File: src/api/queries/blockchainQueries.js [hunk 5]
```diff
@@ -158,3 +158,20 @@ ApiClient_.prototype.fetchRealizedValue = function (slug, from, to) {
...
   return this.conn.graphQLQuery(query, 'realizedValue')
 }
+
+ApiClient_.prototype.fetchHistoricalBalance = function (slug, from, to, address) {
+  var query = {
+    'query': '{\
+       historicalBalance(slug: "' + slug + '",\
+                         from: "' + toUTC_(from) + '",\
+                         to: "' + toUTC_(to) + '",\
+                         interval: "1d"\
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: missing comma here

Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: I thought that I've made a stubbed test for this one, I will make it now, thanks for the heads-up, also for the comma

Comment (kalkov) [reply src/api/queries/blockchainQueries.js]: Seems that commas are not needed, maybe it's better to remove them in a separate PR. But add one here for now.

**Pull request merged**